using-timescaledb/distributed-hypertables.md

```diff
@@ -259,6 +259,22 @@ pushing down the `now()` function, the function is constified on the
 access node so that the resulting timestamp is instead pushed down to
 the data nodes.
 
+### Settings for improving query performance
+
+There are a number of configuration parameters that are important for
+good query performance on a distributed hypertable. These following
+settings apply to the access node.
+
+* `enable_partitionwise_aggregate` should be set to `on`.
+* `jit` should be set to `off` as JIT compilation currently doesn't
+  work well with distributed hypertables.
+* `timescaledb.remote_data_fetcher` should be set to `cursor` (the
```

Comment:
Not clear what the directive is here. Should the user set `timescaledb.remote_data_fetcher` to `cursor` or `rowbyrow`?

Reply:
Yeah, it's up to the user. There are caveats with `rowbyrow`, so not recommended by default. Not sure how